book.rb

```diff
@@ -0,0 +1,9 @@
+class Book
+  attr_accessor :author, :title, :rentals
+
+  def initialize(author, title)
+    @author = author
+    @title = title
+    @rentals = []
+  end
```

Comment:
- [ ] To implement the `has-many` side of the relationship between `Book` and `Rentals`, method for adding rentals should be present in this `Book` class. Something like so;
```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```